daemon/dns/dns.go

```diff
@@ -91,7 +100,8 @@ func (d *DefaultSetter) Unset(iface string) error {
 		if method.IsAvailable() {
 			d.publisher.Publish("unset dns for interface [" + iface + "] using: " + method.Name())
 			if err := method.Unset(iface); err != nil {
-				return fmt.Errorf("unsetting dns with %s: %w", method.Name(), err)
+				log.Println(internal.ErrorPrefix, fmt.Errorf("unsetting dns with %s: %w", method.Name(), err))
+				continue
```

Comment:
Cannot comment at the final return, but if no command is able to unset the DNS shouldn't the function return an error, like `Set`?

Reply:
Errors is written to the log, if any. If we return with error then user will not be able to disconnect from VPN.